cpp/cmake/raft-config.cmake.in

```diff
@@ -0,0 +1,30 @@
+@PACKAGE_INIT@
+
+cmake_minimum_required(VERSION 3.17)
+
+@RAFT_BUILD_DIR_EXPORT_SETTINGS@
+
+# make the bundled find modules in this directory available
+list(PREPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}")
+
+include(CMakeFindDependencyMacro)
+
+include("${CMAKE_CURRENT_LIST_DIR}/raft-targets.cmake")
+
+find_dependency(CUDAToolkit)
+
+if(NOT TARGET raft::spdlog_header_only)
```

Comment:
Looking at this more I don't think that the Thrust logic is needed as I see no usage of the `raft::Thrust` target anywhere